scripts/excel-attachments.rb

```diff
@@ -0,0 +1,25 @@
+# Run using rails runner, EG:
+# PUBLIC_BODY_URL_NAME=url_name bin/rails runner excel-attachments.rb
+#
+
+unless ENV['PUBLIC_BODY_URL_NAME']
+  puts 'Requires PUBLIC_BODY_URL_NAME envrionment variable'
+  exit 1
+end
+
+include Rails.application.routes.url_helpers
+include LinkToHelper
+
+default_url_options[:host] = AlaveteliConfiguration.domain
+
+pb = PublicBody.find_by!(url_name: ENV['PUBLIC_BODY_URL_NAME'])
+all_attachments = FoiAttachment.joins(
+  incoming_message: { info_request: [:public_body] }
+).where(public_bodies: { id: pb })
+attachments_of_interest = all_attachments.where('filename ~* ?', '\.xlsx?$')
```

Comment:
`~*` is "String matches regular expression, case insensitively"

https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-POSIX-REGEXP